MaterialSkin/Controls/MaterialExpansionPanel.cs

```diff
@@ -0,0 +1,607 @@
+using MaterialSkin.Animations;
+using System;
+using System.Configuration;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Text;
+using System.Windows.Forms;
+
+
+namespace MaterialSkin.Controls
+{
+    public class MaterialExpansionPanel : Panel, IMaterialControl
+    {
+
+        #region "Private members"
+
+        private const int _expansionPanelDefaultPadding = 16;
+        private const int _leftrightPadding = 24;
+        private const int _buttonPadding = 8;
+        private const int _expandcollapsbuttonsize = 24;
+        private const int _textHeaderHeight = 24;
+        private const int _headerHeightCollapse = 48;
+        private const int _headerHeightExpand = 64;
```

Comment:
Also, I didn't verify this, but the component seems to have more padding then the other components, check the "padding and margins" values I use on other components, those should be constant trough the lib. I actually need to put them on a helper or something, but that's a refactor for another day, for now, just use the same values I do. I may just be crazy tho, so if I am wrong, just ignore this :P